src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -296,13 +299,20 @@ public void updateOdometry() {
           }
         }
         // Adds the vision measurement for this camera
-        m_poseEstimator.addVisionMeasurement(
-          estimatedPose.estimatedPose.toPose2d(),
-          estimatedPose.timestampSeconds,
-          VisionConstants.kBaseVisionPoseStdDevs.plus(
-            currentEstimatedPoseTranslation.getDistance(closestTagPoseTranslation) * VisionConstants.kVisionPoseStdDevFactor
-          )
-        );
+        if(m_chargeStationAuto){
+          m_poseEstimator.addVisionMeasurement(
+            estimatedPose.estimatedPose.toPose2d(),
+            estimatedPose.timestampSeconds
```

Comment:
This just uses default std devs, it doesn't trust vision "more"

Reply:
Should be switched i think

Reply:
It doesn't use any std devs, so doesn't that make it trust vision completely, like before you added them?